.travis.yml

```diff
@@ -42,11 +42,11 @@ script:
     then
       docker build --squash --no-cache=true -t ballerina/jre8:v1 ./base/base-image
       trivy --exit-code 1 --severity HIGH --no-progress ballerina/jre8:v1
-      mvn clean install -B -V
+      gradle build
     fi
     if [ "$TRAVIS_OS_NAME" = "windows" ]
     then
-      powershell -C 'mvn clean install "-Dmaven.test.skip=true" -B -V'
+      powershell -C 'gradle build -x test'
```

Comment:
Need to pass in the following flags:
` --stacktrace -scan --console=plain --no-daemon`